policybot/cmd/server.go

```diff
@@ -87,7 +88,6 @@ func serverCmd() *cobra.Command {
 		env.RegisterStringVar("GITHUB_OAUTH_CLIENT_SECRET", ca.StartupOptions.GitHubOAuthClientSecret, githubOAuthClientSecret).Get()
 	ca.StartupOptions.GitHubOAuthClientID =
 		env.RegisterStringVar("GITHUB_OAUTH_CLIENT_ID", ca.StartupOptions.GitHubOAuthClientID, githubOAuthClientID).Get()
-	env.RegisterBoolVar("HTTPS_ONLY", ca.StartupOptions.HTTPSOnly, httpsOnly).Get()
```

Comment:
This needs to stay here for now, removing this will require a few other changes including in the k8s deployment YAML files. I'll take care of it.